definitions/divorce/json/CaseEventToFields/CaseEventToFields-payment-by-account-nonprod.json

```diff
@@ -0,0 +1,198 @@
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "02/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorStatementOfTruthPaySubmit",
+    "CaseFieldID": "SolStatementOfReconciliationName",
+    "PageFieldDisplayOrder": 16,
+    "DisplayContext": "MANDATORY",
+    "PageID": "SolStatementOfTruth",
+    "PageLabel": "Statement of truth and reconciliation",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "02/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorStatementOfTruthPaySubmit",
+    "CaseFieldID": "SolStatementOfReconciliationFirm",
+    "PageFieldDisplayOrder": 17,
+    "DisplayContext": "MANDATORY",
+    "PageID": "SolStatementOfTruth",
+    "PageLabel": "Statement of truth and reconciliation",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "01/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorStatementOfTruthPaySubmit",
+    "CaseFieldID": "LabelSolStatementOTruthPara-4",
+    "PageFieldDisplayOrder": 18,
+    "DisplayContext": "READONLY",
+    "PageID": "SolStatementOfTruth",
+    "PageLabel": "Statement of truth and reconciliation",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "02/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorStatementOfTruthPaySubmit",
+    "CaseFieldID": "LabelSolStatementOTruthPara-7",
+    "PageFieldDisplayOrder": 19,
+    "DisplayContext": "READONLY",
+    "PageID": "SolStatementOfTruth",
+    "PageLabel": "Statement of truth and reconciliation",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "02/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorStatementOfTruthPaySubmit",
+    "CaseFieldID": "LabelSolStatementOTruthPara-8",
+    "PageFieldDisplayOrder": 20,
+    "DisplayContext": "READONLY",
+    "PageID": "SolStatementOfTruth",
+    "PageLabel": "Statement of truth and reconciliation",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "02/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorStatementOfTruthPaySubmit",
+    "CaseFieldID": "StatementOfReconciliationComments",
+    "PageFieldDisplayOrder": 21,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "SolStatementOfTruth",
+    "PageLabel": "Statement of truth and reconciliation",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "02/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorStatementOfTruthPaySubmit",
+    "CaseFieldID": "solApplicationFeeInPounds",
+    "PageFieldDisplayOrder": 22,
+    "DisplayContext": "READONLY",
+    "PageID": "SolStatementOfTruth",
+    "PageLabel": "Statement of truth and reconciliation",
+    "PageDisplayOrder": 1,
+    "FieldShowCondition": "D8ReasonForDivorce=\"hiddenfield\"",
```

Comment:
Can you please describe what it does? It's not displayed on this page. It's not displayed on check your answer.

It's `READONLY`. What's the point of it? :)